cogboard-webapp/src/components/BoardForm/index.js

```diff
@@ -8,10 +8,11 @@ import { useFormData } from '../../hooks';
 import { getBoards } from '../../selectors';
 import { createValidationSchema } from './validators';
 import { trimLeadingZeros } from '../helpers';
+import NumberInput from "../widgets/dialogFields/NumberInput";
```

Comment:
Add empty line between components and other imports. Components should be imported as the last ones.

Reply:
done